gnocchi/incoming/redis.py

```diff
@@ -140,15 +140,15 @@ def iter_on_sacks_to_process(self):
         p = self._client.pubsub()
         db = self._client.connection_pool.connection_kwargs['db']
         keyspace = b"__keyspace@" + str(db).encode() + b"__:"
-        pattern = keyspace + self.SACK_PREFIX.encode() + b"*"
+        pattern = keyspace + self._get_sack_name("*").encode()
```

Comment:
this is two different patterns. i assume it didn't matter? just asking because this is probably not tested anywhere iirc.

Reply:
I don't understand the question, and this code path is tested in test_incoming.

Reply:
hmm.. i was under impression greedy code was never tested

my comment was the pattern was  `__keyspace@<db>__incoming*` now it's, `__keyspace@<db>__incoming<number>-*`

i assume it doesn't matter if there's a test.

Reply:
Yeah the new pattern is actually better in this regard. :)

Test: https://github.com/gnocchixyz/gnocchi/blob/master/gnocchi/tests/test_incoming.py#L34-L66